lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -25,6 +25,7 @@ class CampaignSlide extends React.Component {
 
     this.controller.getCampaigns(args).then(campaigns => {
       this.campaigns = campaigns.splice(0, 3); // Grab the first 3 campaigns returned
+      this.forceUpdate();
```

Comment:
You should be storing the campaigns in state if you want React to diff them and re-render.